oio/cli/admin/common.py

```diff
@@ -28,12 +46,13 @@ def patch_parser(self, parser):
         )
 
 
-class ContainerCommandMixin(object):
+class ContainerCommandMixin(ItemCheckCommandMixin):
```

Comment:
For what reason do we need to inherit from `ItemCheckCommandMixin`. I don't get it. This adds a `--depth` argument to commands like `container move`, which makes no sense.